eth/catalyst/api_optimism_test.go

```diff
@@ -34,9 +34,20 @@ func postIsthmus() *params.ChainConfig {
 	return cfg
 }
 
+func postJovian() *params.ChainConfig {
+	cfg := new(params.ChainConfig)
+	// Jovian is post-Holocene on Optimism chains
+	cfg.HoloceneTime = new(uint64)
+	cfg.JovianTime = new(uint64)
+	// Mark as an Optimism chain so IsOptimismJovian is true when JovianTime is active
+	cfg.Optimism = &params.OptimismConfig{}
+	return cfg
+}
+
 var valid1559Params = []byte{0, 1, 2, 3, 4, 5, 6, 7}
 var validExtraData = []byte{0, 1, 2, 3, 4, 5, 6, 7, 8}
 var emptyWithdrawals = make([]*types.Withdrawal, 0)
+var validJovianExtraData = append(append([]byte{1}, valid1559Params...), []byte{0, 0, 0, 0, 0, 0, 0, 0}...) // version=1, 8 bytes params, 8 byte minBaseFee
```

Comment:
```suggestion
var validJovianExtraData = append(append([]byte{1}, valid1559Params...), make([]byte, 8)...) // version=1, 8 bytes params, 8 byte minBaseFee
```
Possibly easier to verify?

Reply:
[f785e13](https://github.com/ethereum-optimism/op-geth/pull/648/commits/f785e13a24cb3216faba37f6c2e1f5d596cabfe5)